core/src/test/scala/org/apache/spark/ContextCleanerSuite.scala

```diff
@@ -245,7 +245,7 @@ class ContextCleanerSuite extends ContextCleanerSuiteBase {
     assert(fs.exists(RDDCheckpointData.rddCheckpointDataPath(sc, rddId).get))
 
     // Test that GC causes checkpoint data cleanup after dereferencing the RDD
-    postGCTester = new CleanerTester(sc, Seq(rddId), Nil, Nil)
+    postGCTester = new CleanerTester(sc, Seq(rddId), Nil, Nil, Seq(rddId))
```

Comment:
@srowen This is just a test code bug. `new CleanerTester(sc, Seq(rddId), Nil, Nil)`  This code is only to ensure that the RDD is cleaned, but checkpoint.  rdd and checkpoint almost simultaneously be cleaned, But there are exceptions, depending on the GC.  The `checkpointCleaned ` ensure the checkpoint is cleaned.